gobblin-utility/src/main/java/gobblin/util/filesystem/FileSystemInstrumentation.java

```diff
@@ -25,9 +27,36 @@
  */
 public class FileSystemInstrumentation extends FileSystemDecorator {
 
+  protected boolean closed = false;
```

Comment:
It is generally an error to call `closed` twice from different threads. However, I think it's easy enough and potentially useful to make this thread safe, so I'll do it.